contract-ui/tabs/overview/page.tsx

```diff
@@ -1,40 +1,256 @@
+      url: "https://blog.thirdweb.com/guides/nft-drop-with-revenue-share/",
+    },
+    { title: "View all", url: "https://blog.thirdweb.com/tag/split/" },
+  ],
+  "token-drop": [
+    {
+      title: "Build An ERC20 Token Claim App in React",
+      url: "https://blog.thirdweb.com/guides/claim-erc20-token-nextjs/",
+    },
+    { title: "View all", url: "https://blog.thirdweb.com/tag/token-drop/" },
+  ],
+  token: [
+    {
+      title: "Create an ERC20 token with TypeScript",
+      url: "https://blog.thirdweb.com/guides/create-your-currency-with-typescript-sdk/",
+    },
+    {
+      title: "Create an ERC20 token with Python",
+      url: "https://blog.thirdweb.com/guides/create-your-currency-with-python-sdk/",
+    },
+    { title: "View all", url: "https://blog.thirdweb.com/tag/token/" },
+  ],
+  vote: [
+    {
+      title: "Build a DAO With a Treasury and a Governance Token",
+      url: "https://blog.thirdweb.com/guides/build-treasury-and-governance-for-your-dao/",
+    },
+    { title: "View all", url: "https://blog.thirdweb.com/tag/vote/" },
+  ],
+  custom: [],
+};
+
+const TEMPLATES = {
+  "edition-drop": [
+    {
+      title: "Edition Drop Minting",
+      url: "https://github.com/thirdweb-example/edition-drop",
+    },
+    {
+      title: "Play-to-Earn",
+      url: "https://github.com/thirdweb-example/play-to-earn-game",
+    },
+    {
+      title: "DAO",
+      url: "https://github.com/thirdweb-example/dao",
+    },
+    {
+      title: "Edition Staking",
+      url: "https://github.com/thirdweb-example/edition-staking-app",
+    },
+  ],
+  edition: [
+    {
+      title: "Karting Game - Unity",
+      url: "https://github.com/thirdweb-example/unity-karting-game",
+    },
+  ],
+  marketplace: [
+    {
+      title: "Marketplace",
+      url: "https://github.com/thirdweb-example/marketplace",
+    },
+    {
+      title: "Unity NFT Loot Boxes",
+      url: "https://github.com/thirdweb-example/unity-nft-lootboxes",
+    },
+    {
+      title: "Marketplace With Signature-based Minting",
+      url: "https://github.com/thirdweb-example/marketplace-with-signature-based-minting",
+    },
+    {
+      title: "Karting Game - Unity",
+      url: "https://github.com/thirdweb-example/unity-karting-game",
+    },
+  ],
+  multiwrap: [
+    {
+      title: "Multiwrap",
+      url: "https://github.com/thirdweb-example/multiwrap",
+    },
+  ],
+  "nft-collection": [],
+  "nft-drop": [],
+  pack: [],
+  "signature-drop": [],
+  split: [],
+  "token-drop": [],
+  token: [],
+  vote: [],
+  custom: [],
+};
+
 export const CustomContractOverviewPage: React.FC<
   CustomContractOverviewPageProps
-> = ({ contractAddress }) => {
+> = ({ contractAddress, contractType }) => {
   const { contract, isSuccess, isError } = useContract(contractAddress);
   const functions = useContractFunctions(contract?.abi as Abi);
   if (!contractAddress) {
     return <div>No contract address provided</div>;
   }
 
+  contractType = "edition-drop";
```

Comment:
oh we're hardcoding this that's why I was getting edition-drop guides on nft-drop :D
```suggestion

```

Reply:
I think @shift4id you had this fixed already, just not pushed up yet?